api/src/db/migrations/v107_refactor_interstate_transport_essential_question.ts

```diff
@@ -0,0 +1,382 @@
+import { v106UserData } from "./v106_add_pet_care_housing_essential_question";
+
+export interface v107UserData {
+  user: v107BusinessUser;
+  profileData: v107ProfileData;
+  formProgress: v107FormProgress;
+  taskProgress: Record<string, v107TaskProgress>;
+  taskItemChecklist: Record<string, boolean>;
+  licenseData: v107LicenseData | undefined;
+  preferences: v107Preferences;
+  taxFilingData: v107TaxFilingData;
+  formationData: v107FormationData;
+  lastUpdatedISO: string | undefined;
+  version: number;
+}
+
+export const migrate_v106_to_v107 = (v106Data: v106UserData): v107UserData => {
+  return {
+    ...v106Data,
+    profileData: {
+      ...v106Data.profileData,
+      isInterstateLogisticsApplicable: v106Data.profileData.industryId == "logistics" ? true : undefined,
+      isInterstateMovingApplicable: v106Data.profileData.industryId === "moving-company" ? true : undefined,
```

Comment:
I don't think this logic is correct... if a user has industry=logistics, they are presented with a radio question for the interstate transport, and some of them may have selected "no".

We need to make use of their previous answer to the question. We can't just use set the answer to "yes" for every logistics industry business